default/etc/e-smith/templates/etc/httpd/conf.d/virtualhosts.conf/30ReverseProxyVhost

```diff
@@ -2,6 +2,7 @@
     use esmith::templates;
     use esmith::ConfigDB;
     my $db = esmith::ConfigDB->open_ro('proxypass') || die("Can't open proxypass db");
+    my $dbc = esmith::ConfigDB->open_ro('configuration');
```

Comment:
Is it required? Should be available as $DB...

Reply:
agreed